lib/utils/stringify.test.ts

```diff
@@ -0,0 +1,38 @@
+import { stringifyValue, stringifySymbol } from './stringify.ts';
+
+describe('stringifySymbol', () => {
```

Comment:
note: with regards to consistency, apart from the usage of `it` vs `test`, sometimes we "call" the function int he `describe` section, but others we don't, like here for example.

```suggestion
describe('stringifySymbol()', () => {
```